### ISSUE TITLE
Paper results replication

### ISSUE THREAD

**Issue by alexcbb** [dido1998/CTRL-O#2]
Hi, thank you for this amazing work. I've tried to launch the training on VG+COCO of the given model and was not able to achieve the same level of results as shown in table 1 of the paper.
Here are my obtained results:
⇒ **mBO**: 21.7.5; **FG-ARI**: 41.6 **Binding-Hits**: 59.9
VS paper results	     
⇒ **mBO**: 27.2; **FG-ARI**: 47.5 **Binding-Hits**: 61.3

Moreover, it seems that masks in Slot-Attention are not good compared to decoders masks:
![Image](https://github.com/user-attachments/assets/85c2c32e-d46c-469e-99d0-9be7f73b5367) (slot-attention)
![Image](https://github.com/user-attachments/assets/74c06eca-3c36-4ae0-b279-33eb9406c8e8) (decoder)

Did this phenomenon happened during your training ?

Thank you for your answer.

**Comment by dido1998** [comment]
Thanks for your interest! The results from the paper which you mention are from a model trained only on COCO. We always use decoder masks for reporting, we did not check the slot attention masks.

**Comment by alexcbb** [comment]
> Thanks for your interest! The results from the paper which you mention are from a model trained only on COCO. We always use decoder masks for reporting, we did not check the slot attention masks.

Hi, thank you for your answer. The results obtained with COCO only are generally worse than the one obtained with VG+COCO which makes me wonder how to properly obtain the results of the table. Do you have maybe a proper permitting to reproduce the announced results ? :)

**Comment by dido1998** [comment]
Did you curate the COCO data using the scripts in the repository? i.e. is it labelled with center of mass information and categories in a way which the code expects or did you use the original coco dataset as is?

**Comment by alexcbb** [comment]
> Did you curate the COCO data using the scripts in the repository? i.e. is it labelled with center of mass information and categories in a way which the code expects or did you use the original coco dataset as is?

Hi, sorry for the late reply but yes I used the script to download and convert the COCO dataset
